Original version does not exist at /usr/ports/games/bomb-her-man
Using /usr/ports/games/bombherman instead
===> Generating patch
===> Viewing diff with more
diff -ruN --exclude=CVS /usr/ports/games/bombherman/Makefile ./Makefile
--- /usr/ports/games/bombherman/Makefile	2013-04-26 16:36:09.000000000 +0200
+++ ./Makefile	2013-05-27 23:26:29.000000000 +0200
@@ -1,4 +1,4 @@
-# Created by: nemysis@gmx.ch
+# Created by: nemysis <nemysis@gmx.ch>
 # $FreeBSD: head/games/bombherman/Makefile 316464 2013-04-24 18:10:30Z ak $
 
 PORTNAME=	bomb-her-man
@@ -62,7 +62,7 @@
 	@(cd ${WRKSRC} && ${SETENV} ${CONFIGURE_ENV} ./autogen.sh)
 
 .if ${PORT_OPTIONS:MNLS}
-	@${REINPLACE_CMD} -e 's|itlocaledir = $$(prefix)/$$(DATADIRNAME)/locale|itlocaledir = $$(prefix)/share/locale|' \
+	@${REINPLACE_CMD} -e 's|$$(prefix)/$$(DATADIRNAME)/locale|$$(prefix)/share/locale|' \
 		${WRKSRC}/po/Makefile.in.in
 .endif
 
@@ -75,7 +75,7 @@
 	${LN} -sf ${PREFIX}/share/icons/hicolor/48x48/apps/${PORTNAME}.png ${PREFIX}/share/pixmaps
 
 .if ${PORT_OPTIONS:MDOCS}
-	${MKDIR} ${DOCSDIR}
+	@${MKDIR} ${DOCSDIR}
 	${INSTALL_DATA} ${PORTDOCS:S|^|${WRKSRC}/|} ${DOCSDIR}
 .endif
 
diff -ruN --exclude=CVS /usr/ports/games/bombherman/pkg-plist ./pkg-plist
--- /usr/ports/games/bombherman/pkg-plist	2013-02-20 21:01:01.000000000 +0100
+++ ./pkg-plist	2013-03-26 04:19:12.000000000 +0100
@@ -61,18 +61,6 @@
 share/icons/hicolor/96x96/apps/bomb-her-man.png
 %%NLS%%share/locale/fr/LC_MESSAGES/bomb-her-man.mo
 share/pixmaps/bomb-her-man.png
-@dirrmtry share/icons/hicolor/96x96/apps
-@dirrmtry share/icons/hicolor/96x96
-@dirrmtry share/icons/hicolor/72x72/apps
-@dirrmtry share/icons/hicolor/72x72
-@dirrmtry share/icons/hicolor/64x64/apps
-@dirrmtry share/icons/hicolor/64x64
-@dirrmtry share/icons/hicolor/48x48/apps
-@dirrmtry share/icons/hicolor/48x48
-@dirrmtry share/icons/hicolor/32x32/apps
-@dirrmtry share/icons/hicolor/32x32
-@dirrmtry share/icons/hicolor
-@dirrmtry share/icons
 @dirrm %%DATADIR%%/graphics/scores/2
 @dirrm %%DATADIR%%/graphics/scores/1
 @dirrm %%DATADIR%%/graphics/scores
===> Done
